### PR TITLE
Boost 1.59.0

### PR DESCRIPTION
**Pull request by mikemcquaid**
Another try from #43994.

### CODE REVIEW

**Comment by DomT4** [comment]
I nearly went and closed this with a slightly grumpy message about checking prior PRs before submitting new ones :sweat_smile:.

**Comment by mikemcquaid** [comment]
@DomT4 BRING IT :laughing:

**Comment by DomT4** [comment]
:stuck_out_tongue_winking_eye:

**Comment by mikemcquaid** [comment]
CI is :green_apple:. Will try to build `brew uses boost` locally to verify what else needs done.

**Comment by mikemcquaid** [comment]
Think I've fixed all the issues here.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

File: Library/Formula/boost-python.rb [hunk 1]
```diff
@@ -42,6 +33,14 @@ class BoostPython < Formula
   def install
     ENV.universal_binary if build.universal?
 
+    if stable?
```
Comment (DomT4) [inline Library/Formula/boost-python.rb]: Should this be `build.stable?` or?

**Comment by DomT4** [comment]
Did these get checked locally or do they need bumping after this is merged?

```
homebrew/fuse/encfs
homebrew/fuse/ori
homebrew/head-only/kakoune
homebrew/versions/ledger26
homebrew/versions/libmongoclient-legacy
homebrew/versions/mapnik2
homebrew/versions/mongodb26
homebrew/versions/thrift090
homebrew/x11/inkscape
homebrew/x11/scantailor
```

**Comment by mikemcquaid** [comment]
@DomT4 Checked all those in versions.

**Comment by DomT4** [comment]
Nice. Thanks Mike.

**Comment by mikemcquaid** [comment]
Others look good too :+1:

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by tdsmith** [comment]
Thanks Mike!

**Comment by mikemcquaid** [comment]
@tdsmith My (almost) pleasure!

**Pull request closed without merging**